azure-pipelines-1.yml

```
@@ -0,0 +1,19 @@
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- master
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- script: echo Hello, world!
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 2 but found 0

(indentation)